examples/WORKSPACE

```diff
@@ -0,0 +1,29 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+RULES_JVM_EXTERNAL_TAG = "2.2"
+RULES_JVM_EXTERNAL_SHA = "f1203ce04e232ab6fdd81897cf0ff76f2c04c0741424d192f28e65ae752ce2d6"
+
+http_archive(
+     name = "rules_jvm_external",
+     strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_TAG,
+     sha256 = RULES_JVM_EXTERNAL_SHA,
+     url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
+ )
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+load("@rules_jvm_external//:specs.bzl", "maven")
+
+maven_install(
+     artifacts = [
+	"info.picocli:picocli:3.9.5",
+	],
+	repositories = [
+         "https://maven.google.com",
+         "https://repo1.maven.org/maven2",
+     	],
+)
+
+local_repository(
+    name = "builder",
+    path = "/Users/kvijayan/git/bazel-java-builder-template",
```

Comment:
Can you try a relative path, i.e ".."?

Reply:
We can try this tomorrow after getting it to run, just leaving it as is before execution.